experimental/IntersectionTheory/docs/src/AbstractVarieties.md

```diff
@@ -60,7 +60,7 @@ degeneracy_locus(F::AbstractBundle, G::AbstractBundle, k::Int; class::Bool=false
 ```
 
 !!! note
-    Products and blowups are described elsewhere.
+    Products and blow_ups are described elsewhere.
```

Comment:
Thanks for renaming the method `blowup(...)` to `blow_up(...)`. 
It seems that in some places such as here, the word blowup was changed as well, where it is not a function name but a just mathematical terminology. Maybe these changes were unintended?

Reply:
@simonbrandhorst thanks; reversed with [1d22cc0](https://github.com/oscar-system/Oscar.jl/pull/4526/commits/1d22cc098472bca3016ccb61f984cd035accaf2d)